Tests/SentryTests/Integrations/Session/SentryAutoSessionTrackingIntegrationTests.swift

```diff
@@ -4,7 +4,7 @@ import XCTest
 class SentryAutoSessionTrackingIntegrationTests: XCTestCase {
 
     func test_AutoSessionTrackingEnabled_TrackerInitialized() {
-        let sut = SentryAutoSessionTrackingIntegration()
+        let sut = SentryAutoSessionTrackingIntegration(crashWrapper: TestCrashWrapper())
```

Comment:
Having to provide the crash wrapper for so many tests not using it is another sign to me we shouldn't add it to the default constructor as discussed in https://github.com/getsentry/sentry-cocoa/pull/2942#discussion_r1173430878